src/main/java/org/diatliuk/bookstore/model/Role.java

```diff
@@ -0,0 +1,23 @@
+package org.diatliuk.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.diatliuk.bookstore.enums.RoleName;
+
+@Data
+@Entity
+@Table(name = "roles")
```

Comment:
```suggestion
@Table(name = "role")
```

Reply:
I guess that it's not a problem, and it depends on naming conventions in a team